packages/snaps-utils/src/manifest/validation.ts

```diff
@@ -268,7 +268,7 @@ export type SnapPermissions = InferMatching<
   InitialPermissions
 >;
 
-export const SnapAuxilaryFilesStruct = array(string());
+export const SnapAuxiliaryFilesStruct = array(string());
```

Comment:
Thanks, this makes sense to me. For backwards compatibility sake though, can you add an alias of the fixed variable and mark it `@deprecated`?